doc/source/cluster/kubernetes/user-guides/kubectl-plugin.md

```diff
@@ -0,0 +1,91 @@
+(kubectl-plugin)=
+
+# Use Kubectl Plugin (Alpha Feature)
+
+Today it is incredibly challenging for data scientists and AI researchers unfamiliar with Kubernetes to start using Ray on Kubernetes. However, running Ray on Kubernetes is advantageous or necessary in certain environments.
```

Comment:
I think we can remove this sentence. This was important context for why we built it, but we don't need to include it in the guide. 
```
Starting from KubeRay v1.2.2, we have introduced a `kubectl ray` plugin to simplify common workflows when deploying Ray on Kubernetes. The plugin is designed to help users who are not familiar with Kubernetes to run Ray on Kubernetes with ease.
```

Reply:
Done in https://github.com/ray-project/ray/pull/47851/commits/e802c4ba0bb1ec5b5624ef664180b73e5e72426b